torch/nn/modules/module.py

```diff
@@ -1426,6 +1429,34 @@ def _register_load_state_dict_pre_hook(self, hook, with_module=False):
         self._load_state_dict_pre_hooks[handle.id] = hook
         return handle
 
+    def register_load_state_dict_post_hook(self, hook):
+        r"""Registers a post hook to be run after module's ``load_state_dict``
```

Comment:
nit: the second sentence just repeats the first one. We can remove it.

Or if we want to follow the other hooks style, add a paragraph break in between the two.